contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,296 @@
+---
+date: 2023-02-07
+title: "Retention vs Churn Rate: A complete guide to churn analysis"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - andy-vandervell
+featuredImage: ../images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+category: Product growth
+tags:
+  - Product analytics
+  - Product metrics
+  - Guides
+  - Explainers
+---
+
+You know what's cooler than gaining new customers? Retaining them.
+
+In this guide to customer churn analysis, I'm exploring churn rate and retention rate:
+- How are they different?
+- How do you measure them? 
+- Which one would win a fight?<sup>*</sup>
+
+These and other useful questions (like [how to reduce churn](#how-to-improve-retention-and-reduce-churn) will be answered. Let's get to it.
+
+<sup>(*strictly speaking, they'd cancel each other out... read the next section to understand why)</sup>
+
+## What's the difference between retention rate and churn rate?
+
+Not much, really: **churn rate** is the percentage of customers who **stop using** your product during a given period of time; **retention rate** is the percentage of **existing customers** who **continue using** your product. 
+
+In other words, if you have 1,000 customers and 100 of them cancel their subscription, your monthly churn rate would be 10% and your retention rate would be 90%. That's it.
+
+Simple, right? Yes, but these definitions ignore three important points:
+
+1. There are numerous ways to measure and analyze them
+1. Just knowing what they are won't help you improve them
+1. While both are important, in my view **retention rate** is more useful (most of the time)
+
+I'll explore these points further as we go. See [Takeaways on churn analysis](#takeaways-on-churn-analysis) for a TL;DR.
+
+## Understanding churn rate
+
+Here's the simplest formula for churn rate:
+
+> **Customer churn rate** = (Churned customers during a specific period / Total customers at the start of that period) x 100
```

Comment:
```suggestion
> **Customer churn rate** = Churned customers during a specific period / Total customers at the start of that period x 100
```
Based on order of operations, you don't need the brackets here.

Reply:
I think I prefer to keep them in. Makes it a little easier parse.